sdk/rust/examples/backend/src/main.rs

```diff
@@ -0,0 +1,46 @@
+mod configuration;
+
+use eyre::Result;
+use dagger_sdk::{Query, File, Container};
+use clap::Parser;
+
+use configuration::Configuration;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let client = dagger_sdk::connect().await?;
+    let build = build_backend(&client).await;
+    let image = build_prod_image(&client, build).await;
+    let image_reference = push_image(image).await?;
+    println!("Image published at: {}", image_reference);
+    Ok(())
+}
+
+async fn build_backend(client: &Query) -> File {
+    let backend_directory = client.host().directory("axum-backend");
+    client
+        .container()
+        .from("rust:alpine")
```

Comment:
consider pinning the version of the image, otherwise we may silently fail builds